Workbooks/SDKStats/SDKStats.workbook

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## SDK Stats\n---\nThe SDK Stats workbook provides a clear visualization of success and failure metrics for your application. It leverages the Azure Monitor SDK to track and chart two key custom metrics: preview.item.success.count and preview.item.dropped.count. This workbook helps teams monitor operational health, identify trends, and quickly detect anomalies in customer-facing processes by displaying these metrics over time in an intuitive line chart format.\n"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "customMetrics\n| where name in (\"preview.item.dropped.count\", \"preview.item.success.count\")\n| summarize total_count = sum(value) by name, bin(timestamp, 1m)\n| extend display_name = case(\n    name == \"preview.item.success.count\", \"success count\",\n    name == \"preview.item.dropped.count\", \"dropped count\",\n    name\n)\n| project timestamp, name = display_name, total_count\n| render timechart",
+        "size": 1,
+        "title": "Telemetry Export Success Count",
+        "timeContext": {
```

Comment:
instead of hardcoding a time range, shoudl there be a time range parameter users can chose from?

the query could then adjust the bin in the query with `bin(timestamp, {TimeRange:grain})` instead of being hardcoded to 1m?

Reply:
Updated to include a time-picker and included the `TimeRange:grain` to auto-select a granularity for the timeseries.